Shiprekt/Content/TiledObjects.Generated.xml

```diff
@@ -1,11 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
 <objecttypes xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:xsd="http://www.w3.org/2001/XMLSchema">
-  <objecttype name="Bird" />
+  <objecttype name="Bird">
+    <property name="MaxFlightSpeed" type="float" default="30" />
+    <property name="MaxTurnRate" type="float" default="1.57" />
+    <property name="MaxSecondsToFlightStateChange" type="float" default="3" />
+    <property name="MinFlightSpeed" type="float" default="8" />
+    <property name="MinBirdAltitude" type="float" default="20" />
+    <property name="MaxBirdAltitude" type="float" default="200" />
+  </objecttype>
   <objecttype name="Bullet">
     <property name="BulletSpeed" type="float" default="300" />
     <property name="ReloadTimeInSeconds" type="float" default="1" />
     <property name="BulletDistance" type="float" default="200" />
-    <property name="DamageToDeal" type="float" default="20" />
+    <property name="DamageToDeal" type="int" default="0" />
```

Comment:
I don't know what happened here. I certainly didn't plan to change this. I'll see if I can revert this.

Reply:
I reverted this manually.